runtime/j9vm/jvm.c

```diff
@@ -4478,11 +4507,18 @@ JVM_RaiseSignal(jint sigNum)
 {
 	jboolean rc = JNI_FALSE;
 	J9JavaVM *javaVM = (J9JavaVM *)BFUjavaVM;
+	BOOLEAN isSpecialSignal = isSignalSpecial(sigNum);
 
 	Trc_SC_RaiseSignal_Entry(sigNum);
 
-	if (J9_ARE_ALL_BITS_SET(javaVM->sigFlags, J9_SIG_XRS)
-			&& isSignalSpecial(sigNum)) {
+	if (J9_ARE_ANY_BITS_SET(javaVM->sigFlags, J9_SIG_XRS | J9_SIG_XRS_ASYNC)
+			&& (isSpecialSignal
+#if defined(SIGQUIT)
+			|| (SIGQUIT == sigNum)
```

Comment:
Should this check isSignalUsedByVM()?

Reply:
According to https://github.com/ibmruntimes/openj9-openjdk-jdk9/blob/openj9/jdk/test/sun/misc/SunMiscSignalTest.java, 

`SIGHUP`, `SIGINT`, `SIGTERM` can't be registered, invoked or raised if `-Xrs` is present.
`SIGQUIT` can't be registered, invoked or raised at all times. 

Updated the definition of JVM_RaiseSignal to reflect this behavior.